packages/trader/src/AppV2/Containers/Positions/positions-content.tsx

```diff
@@ -8,7 +8,7 @@ import { ContractCardList, ContractCardsSections } from 'AppV2/Components/Contra
 import { ContractTypeFilter, TimeFilter } from 'AppV2/Components/Filter';
 import TotalProfitLoss from 'AppV2/Components/TotalProfitLoss';
 import { filterPositions, getTotalPositionsProfit, TAB_NAME } from '../../Utils/positions-utils';
-import { TReportsStore, useReportsStore } from '../../../../../reports/src/Stores/useReportsStores';
+import { TReportsStore, useReportsStore } from '@deriv/reports/src/Stores/useReportsStores';
```

Comment:
this looks like has happened because of linting, this change does it work? yeah it works the page loads but in position page when we navigate to closed contract the page breaks. Not related to this change but we need to check why is it breaking

Reply:
Linter is making unnecessary changes 🤦🏼